src/client/src/ui/spotTile/components/PriceButton.tsx

```diff
@@ -34,17 +34,17 @@ const Box = styled('div')`
 `
 
 const DirectionLabel = styled(Box)`
-  color: ${({ theme: { text } }) => text.textMeta};
+  opacity: 0.59;
   margin: 0 0 0.125rem 0;
   font-size: 0.625rem;
 `
 
 const Big = styled(Box)`
-  font-size: ${p => p.theme.fontSize.h4};
+  font-size: 0.8125rem;
 `
 
 const Pip = styled(Box)`
-  font-size: ${p => p.theme.fontSize.h2};
+  font-size: 2.125rem;
```

Comment:
That is one verbose font size. (no action needed)

Reply:
Except setting a line-height and making sure `2rem` doesn't work just as well — since the marginal value of `.125rem` is … well, increasingly marginal.